lms/djangoapps/edraak_forus/admin.py

```diff
@@ -0,0 +1,53 @@
+"""
+Django admin page for ForUs models
+"""
+from django.contrib import admin
+from django.core.urlresolvers import reverse
+from django.utils.html import escape
+from openedx.core.djangolib.markup import HTML
+
+from student.models import UserProfile
+
+from edraak_forus.models import ForusProfile
+from edraak_forus.decorators import setfuncattr
+
+
+class ForusProfileAdmin(admin.ModelAdmin):
+    """Admin for ForUs profile."""
+    list_display = ('edraak_user', 'edraak_profile', 'email', 'date_created')
+    readonly_fields = ('user', 'date_created')
+    search_fields = ('user__email', 'user__username')
+
+    @setfuncattr('allow_tags', True)
+    def edraak_user(self, profile):
+        """
+        Return link to user
+        """
+        user = profile.user
+        return HTML(u'<a href="{url}">{name}</a>').format(
+            name=escape(user.username),
+            url=reverse('admin:auth_user_change', args=[user.pk]),
+        )
+
+    @setfuncattr('allow_tags', True)
+    def edraak_profile(self, profile):
+        """
+        Returns link to Edraak profile
+        """
+        try:
+            edraak_profile = UserProfile.objects.get(user=profile.user)
+        except UserProfile.DoesNotExist:
+            return None
+
+        return HTML(u'<a href="{url}">Edraak Profile</a>').format(
+            url=reverse('admin:student_userprofile_change', args=[edraak_profile.pk]),
```

Comment:
You might want to use `Text()` for the format params like `url`, this is especially important because the linter won't check for this outside `*.html` files.